plugins/reports/counter/index.php

```diff
@@ -16,8 +16,13 @@
  *
  */
 
+// Because of the use of Namespaces, this plugin now requires PHP 5.3 or better
+if (version_compare(PHP_VERSION, '5.3.0') >= 0) {
```

Comment:
I suggest we limit the amount of time we spend on obsolete hosting environments, even in OJS 2.x, especially in plugins. There are no perfect solutions for some of this, and our long-term solution is OJS 3 plus modern coding standards. I'm content to go ahead with @ctgraham's proposal.